src/components/ConnectionsHeader/index.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Header, ConnectionContainer, ConnectionLabel } from './styled';
+import { useReading } from '../../logic/useReading';
+import { ALARMS } from './icons';
+
+function updateAlarmsUI(alarms: string[]) {
+  return Object.keys(ALARMS).map((warning) => {
+    let connectionStatus = alarms.includes(warning);
+
+    return (
+      <ConnectionContainer key={warning}>
+        {ALARMS[warning].icon(connectionStatus)}
```

Comment:
is it better to not have each of these statuses as a separate component? One which contains the icon and message? The `ConnectionsHeader` will contain all of these. React recommends break down into as many components as possible

Reply:
I looked into it but I dont the see the depth and room for dividing it into to more components, ofcourse we can but I dont feel the need because of the nature of the component, we may not use them outside of this, if you think we can make use then let me know.

Reply:
I do think it's better to have them as components as we could move them out from there into the header bar etc. 

However, it's not a requirement for your task so keep it as-is for now. Resolving.